neonvm/runner/main.go

```diff
@@ -359,19 +417,183 @@ func calcDirUsage(dirPath string) (int64, error) {
 	return size, nil
 }
 
-func createSwap(diskName string, diskPath string, diskSize *resource.Quantity) error {
-	if diskSize == nil {
-		return errors.New("diskSize should be specified")
+const (
+	swapDiskClusterSize string = "2M"
+	// Note: we use an offset of 2MiB here in order to match the cluster_size=2M that we give to
```

Comment:
qcow2 does caching based on subdividing the disk into clusters (size determined by cluster_size here). I haven't done any testing on this, but I _expect_ that aligning the partition start to equal the cluster size is a safe bet, but probably doesn't make much difference in practice (because it's already going to be 1 MiB-aligned). Mostly just there because we have to pick some number, and it might as well equal the cluster size.

Details on qcow2 caching here if you're curious: https://github.com/qemu/qemu/blob/master/docs/qcow2-cache.txt

Reply:
Thx for explaining, suggested a comment change